packages/vscode-extension/src/builders/BuildCache.ts

```diff
@@ -37,30 +37,31 @@ function makeCacheKey(platform: DevicePlatform, appRoot: string) {
 }
 
 export class BuildCache {
-  private readonly cacheKey: string;
-
-  constructor(private readonly platform: DevicePlatform, private readonly appRoot: string) {
-    this.cacheKey = makeCacheKey(platform, appRoot);
-  }
+  constructor(private readonly appRootFolder: string) {}
 
   /**
    * Passed fingerprint should be calculated at the time build is started.
    */
-  public async storeBuild(buildFingerprint: string, build: BuildResult) {
+  public async storeBuild(buildFingerprint: string, build: BuildResult, platform: DevicePlatform) {
```

Comment:
can't we use `buiild.platform` here instead of providing platform as extra argument?